### ISSUE TITLE
[Arcs] [FAQ ready] Peace Accord

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#88]
https://cards.ledergames.com/card/ARCS-F1511

> Q: If in a cluster, two players each control one system, and a third player controls two. Can the two players consent to flip the system to Peace without the consent of the third player?
> A: Yes, because they control a total of two systems.

**Comment by jdyearsley** [comment]
Yup, this is a straightforward interpretation of the literal rule—it uses the number two rather than saying "the most systems." Have there been discussions arguing to the contrary?

**Comment by turmoilbyrd** [comment]
> Yup, this is a straightforward interpretation of the literal rule—it uses the number two rather than saying "the most systems." Have there been discussions arguing to the contrary?

No I think most of the community agree to this understanding. I was actually the one who misunderstood it and thought the player who controls "the most system" gets a say.
